HHS/Head-Start-TTADP
# [TTAHUB-731] Next Steps Redesign

## Description of change

Redesign for AR Next Steps page.

## How to test

Next Steps should match the new design linked on https://ocio-jira.acf.hhs.gov/browse/TTAHUB-731.

- Can create Save and Submit a report.
- Can Add/Update/Remove steps.
- Failure to populate a step(s) on blur should alert the user.
- Failure to populate a step(s) on submit should alert the user.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-731


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
